src/components/Dashboard/FilingHistoryList/filings/LimitedRestoration.vue

```diff
@@ -36,7 +35,7 @@ export default class LimitedRestoration extends Vue {
 
   /** The legal name from the restoration filing. */
   get legalName (): string {
-    return this.filing.data?.restoration?.toLegalName || '[unknown]'
+    return this.filing.data?.restoration?.legalName || '[unknown]'
```

Comment:
The property is `legalName` not `toLegalName`.

Line: https://github.com/JazzarKarim/business-filings-ui/blob/d0b5ec751455746f56bb5043f4794a558ede173b/src/interfaces/api-filing-interface.ts#L99

Reply:
Please attach a screenshot of this in action.

Reply:
Screenshot: 
![limited unklnown](https://github.com/bcgov/business-filings-ui/assets/122301442/4d188d86-c39d-4e63-8614-bc2e07621bcf)


I've reverted this change and it's working well. I did a global search and toLegalName doesn't exist. Not sure why does it work.

Reply:
Unfortunately the Legal API doesn't follow a schema when providing data, so what you see here is a mismatch between ApiFilingIF and what we're actually getting back from the API.

Please fix the IF. Thanks :)

Reply:
Fixed, thanks.